components/Questionnaire/index.vue

```diff
@@ -121,9 +121,11 @@ export default {
       isLevelOne: true,
       isLevelTwo: false,
       isLevelThree: false,
+      isLevelFour: false,
       showLevelOne: true,
       showLevelTwo: false,
       showLevelThree: false,
+      showLevelFour: false,
```

Comment:
unused code?

Reply:
Yes, it's not used and the changes [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/94/commits/ae51bc0772cac27f7c9499339331bbb522f935da)